qa/common/gen_qa_model_repository

```diff
@@ -512,7 +515,7 @@ chmod -R 777 $FORMATDESTDIR
 python3 $SRCDIR/gen_qa_trt_data_dependent_shape.py --models_dir=$DATADEPENDENTDIR
 chmod -R 777 $DATADEPENDENTDIR
 # Make shared library for custom Hardmax plugin.
-(git clone -b release/10.0 https://github.com/NVIDIA/TensorRT.git && \
+(git clone -b release/$(echo ${TRT_VERSION} | cut -d '.' -f 1-2) https://github.com/NVIDIA/TensorRT.git && \
```

Comment:
Added the TRT_VERSION=${TRT_VERSION:-10.5.0.18} above. This can be also included in CI as env variable [here](https://gitlab-master.nvidia.com/dl/dgx/tritonserver/-/blob/master/ci/server/models/build-models-ci.yml?ref_type=heads#L42).

Reply:
My intention here was to keep us away from tracking that version.

I will leave it up to you, bear in mind that next release is 24.11 will use TensorRT version 10.6.0.22

Reply:
Sounds good @mc-nv. I prefer your changes over mine. Can you help create a new ticket and commit these changes. Currently I have not verified the value of com.nvidia.tensorrt.version from docker inspect in a pipeline. Additionally, please note that these changes are similar to those used for generating models in version 24.10.